src/components/CarouselCard/__tests__/animationButton.spec.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import AnimationButtom from '../animationButton';
+
+describe('<AnimationButtom />', () => {
+    it('should set the right assistive text', () => {
+        const component = mount(<AnimationButtom isAnimationPaused />);
+
+        expect(component.find('span').text()).toBe('Start auto-play');
```

Comment:
This is introspection. Remember each component is a black box. If we change the AssistiveText inside the ButtonIcon to use a div instead of span then this fail. Here you only check that the assistiveText prop of ButtonIcon is set to Start auto-play.